src/hotspot/share/opto/superword.cpp

```diff
@@ -1472,60 +1487,191 @@ void SuperWord::combine_pairs_to_longer_packs() {
 #endif
 }
 
-void SuperWord::split_packs_longer_than_max_vector_size() {
-  assert(!_packset.is_empty(), "packset not empty");
-  DEBUG_ONLY( int old_packset_length = _packset.length(); )
+// Split pack according to task. Return true if any change was made, else false.
+SuperWord::SplitStatus
+SuperWord::split_pack(const char* split_name, Node_List* pack, SplitTask task) {
```

Comment:
Style. Please, put return type on the same line as method. Yes, the line will be bigger but I prefer to have arguments on separate line than the return type.

Reply:
Fixed. Used to have this code-style in my previous company :see_no_evil: